app/views/govuk_publishing_components/components/docs/error_message.yml

```diff
@@ -15,9 +15,22 @@ examples:
     data:
       text: "Please enter your National Insurance Number"
       id: "error_id"
+  with_dir_attribute:
```

Comment:
Nitpick: I'd put this example beneath the `with_items` example, as `with_items` feels like more of a prominent use case.

Reply:
Sounds like a fair point. Have changed that. 👍